src/main/kotlin/frc/robot/commandGroups/CommandGroupsConstants.kt

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commandGroups
+
+import com.pathplanner.lib.util.GeometryUtil
+import edu.wpi.first.math.geometry.Translation2d
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.Measure
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.Velocity
+import frc.robot.Constants
+
+object CommandGroupsConstants {
+    val HOOD_ANGLE_SUPER_POOP: Measure<Angle> = Units.Degrees.of(110.0)
+    val SHOOTER_VELOCITY_SUPER_POOP: Measure<Velocity<Angle>> = Units.RotationsPerSecond.of(77.07)
+    val CONVEYOR_VELOCITY_SUPER_POOP: Measure<Velocity<Angle>> = Units.RotationsPerSecond.of(60.0)
+
+    private val SUPER_POOP_TRANSLATION_BLUE = Translation2d()
+
+    val SUPER_POOP_TRANSLATION: Translation2d
+        get() =
+            if (Constants.isRed) GeometryUtil.flipFieldPosition(SUPER_POOP_TRANSLATION_BLUE)
+            else SUPER_POOP_TRANSLATION_BLUE
```

Comment:
We do this in quite a few places. Perhaps extract it to another function that flips if needed?

Reply:
62182daf74cf7a0ab44bd6bd17cd03b977b87166